src/main/resources/hudson/security/table.js

```diff
@@ -238,3 +246,9 @@ Behaviour.specify(".global-matrix-authorization-strategy-table TR.permission-row
     e.setAttribute('data-checked', 'true');
   }
 });
+
+document.addEventListener('DOMContentLoaded', function () {
+  setTimeout(function () {
+    window.registerTooltips();
+  }, 500);
```

Comment:
quite hacky but haven't managed to find a way yet to call this after all inputs have had their javascript applied.

the previously implementation was being called ~220 times with ~5 or so entries. which is why it was so slow